pkg/cluster/arooperator.go

```diff
@@ -53,3 +53,7 @@ func (m *manager) ensureAROOperatorRunningDesiredVersion(ctx context.Context) (b
 func (m *manager) renewMDSDCertificate(ctx context.Context) error {
 	return m.aroOperatorDeployer.RenewMDSDCertificate(ctx)
 }
+
+func (m *manager) restartAROOperatorMaster(ctx context.Context) error {
```

Comment:
Unit testing didn't make sense to me here because the ARO operator deployer's `Restart` method basically just calls already-tested code in a loop, but I added an E2E test. Let me know what you think.